File: Practice27/practice27.rb
puts("put yo id.")
input_id = gets.chomp()
 
def login(_id)
members = ['ab', 'cd', 'ef']
  for member in members do
      if member == _id
          return true
      end
  end
  return false
end
 
if login(input_id)
  puts('Hello, '+input_id)
else
  puts('Who are you?')
end

File: Practice27/practice27.py
input_id = input("put yo id.\n")
def login(_id):
    members = ['ab', 'cd', 'ef']
    for member in members:
        if member == _id:
            return True
    return False
if login(input_id):
    print('Hello, '+input_id)
else:
    print('Who are you?')